src/function/chatPostMessage.js

```diff
@@ -0,0 +1,40 @@
+import https from "https";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+const TOKEN = process.env.LINE_ACCESS_TOKEN;
+
+const HEADERS = {
+	"Content-Type": "application/json",
+	Authorization: `Bearer ${TOKEN}`,
+};
+
+// Send four-day weather forecast information to user
+export function chatPostMessage(WeatherForecastForFourDays, ReplyToken) {
+	const Body = JSON.stringify({
```

Comment:
```suggestion
    const body = JSON.stringify({
```

Reply:
Are you using prettier? it seems empty space is too much.

Reply:
Yes, I am. I guess because of setting?

Reply:
I fixed it